### PR TITLE
Add some documentation and update the list of available icons

### PR DESCRIPTION
**Pull request by laviniaclare**

### CODE REVIEW

File: stories/icons.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+```
+<Icon name="cr-logo" />
+```
+
+The most comprhensive list of icons available in arbor is in fort awesome: https://fortawesome.com/kits/27af4258/icons
+
+## Adding new icons
+
+New icons can be added to the fort awesome arbor kit by either uploading an icon, or picking from various icons already available on fort awesome
+https://fortawesome.com/kits/27af4258
```
Comment (xxx) [inline stories/icons.md:14]: https://github.com/CatchRelease/catchandrelease/wiki/FortAwesome has instructions for this step as well

File: stories/icons.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+### Update arbor
+
+1. Click on "Add" for the icon section
+2. Select or upload an icon
+3. Hit the "Publish" button in fort awesome to publish the change to the arbor kit
+
+### Update the catchandrelease repo
+
+1. Download the updated arbor kit by clicking "Download" (under the publish button)
+2. Replace the icons files in `apps/assets/fonts/arbor-icons` with the newly generated copeies
```
Comment (xxx) [inline stories/icons.md:25]: typo here (`copeies`)

File: stories/icons.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+3. Add the SVG file for the new icon to `app/assets/fonts/arbor-icons/svg`
+4. Add the new icon name to `app/assets/stylesheets/arbor/_icons.sass`
```
Comment (xxx) [inline stories/icons.md:27]: just adding a note to keep this in order of the icons listed at fortawesome

File: stories/icons.md [hunk 4]
```diff
@@ -1 +1,27 @@
-TODO -- Add some information on how this component should be used.
+## Code examples
+
+### Most basic
+
+```
+<Icon name="cr-logo" />
+```
+
+The most comprhensive list of icons available in arbor is in fort awesome: https://fortawesome.com/kits/27af4258/icons
```
Comment (pvenable) [inline stories/icons.md:9]: Typo here: `comprhensive` -> `comprehensive`

Comment (xxx) [inline stories/icons.md:9]: typo here (`comprhensive`)

**Review by xxx — APPROVED** [review]
I think the various proper names should be capitalized, to make it a bit clearer, but lgtm otherwise

**Pull request merged**